src/View/mainpage/DrawROIWindow/Drawing.py

```diff
@@ -132,8 +138,14 @@ def _display_pixel_color(self):
             self.according_color_dict[(x_coord, y_coord)] = colors
 
         points = get_pixel_coords(self.according_color_dict, self.rows, self.cols)
+
         self._set_color_of_pixels(points)
         self.refresh_image()
+        if len(points) < 1:
+            return False
+        else:
+            return True
```

Comment:
this could be simplified, dunno if it would be better to read though
return (len(points) > 1)

Reply:
could also do `return False if len(points) < 1 else True` or re-order if needed